From 8d560d598df40bf7b10cd19336b8f87653be0d1b Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Mon, 14 Aug 2017 18:58:24 +0000
Subject: [PATCH] MANTA-2500 muskie default content length is not one size fits
 all Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com> Approved by:
 Jordan Hendricks <jordan.hendricks@joyent.com>

---
 config/services/webapi/service.json.coal | 2 +-
 config/services/webapi/service.json.lab  | 3 +++
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/config/services/webapi/service.json.coal b/config/services/webapi/service.json.coal
index 5677ff1..fd56d86 100644
--- a/config/services/webapi/service.json.coal
+++ b/config/services/webapi/service.json.coal
@@ -1,5 +1,5 @@
 {
 	"metadata": {
-		"MUSKIE_IGNORE_SIZE": true
+		"MUSKIE_DEFAULT_MAX_STREAMING_SIZE_MB": 5120
 	}
 }
diff --git a/config/services/webapi/service.json.lab b/config/services/webapi/service.json.lab
index 9154a7f..84eceff 100644
--- a/config/services/webapi/service.json.lab
+++ b/config/services/webapi/service.json.lab
@@ -1,5 +1,8 @@
 {
 	"params": {
 		"ram": 1536
+	},
+	"metadata": {
+		"MUSKIE_DEFAULT_MAX_STREAMING_SIZE_MB": 5120
 	}
 }
-- 
2.21.0

